typescript-local-storage/data.ts

```diff
@@ -0,0 +1,29 @@
+/* exported todos */
+interface Todo {
+  todoId: string;
+  task: string;
+  isCompleted: boolean;
+}
+
+let todos: Todo[] = [];
```

Comment:
Rather than assigning an empty array, you should be assigning the result of calling `readTodos`